src/layouts/navigation.ts

```diff
@@ -329,15 +330,31 @@ export const getNavigation = (section) => {
           ],
         },
         {
-          title: "Indexers",
+          title: "Tutorials",
```

Comment:
You are removing the Indexers from the navigation, which I agree with (see #1203 ), but this should be a separate action as the content should be cleaned up as well.